docs/proposals/ose-rebase.md

```diff
@@ -256,12 +253,13 @@ git push origin ${CURRENT_BRANCH} -f refs/tags/${last_tag}
 
 Common conflicts can rise when new changes step on branding or tooling code. Manual
 resolution is the only thing we can do in such cases. Today when a conflict occurs, it is
-resolved manually ie. master is updated with the conflict resolved and a new build is
-kicked off. This is not going to be possible with the suggested process because the
-changelog for tito needs to be constructed based on the previous and the current HEAD of
-Origin that got pulled in with the rebase. All the steps above need to run manually and
-then the build can be restarted. This also means that the build process is idempotent and
-it will skip the above part.
+resolved manually by simply using `git merge`, resolving any conflicts, and then kick off
+a new build. The process of updating master with resolved conflicts needs to slightly
+change because the changelog for tito needs to be constructed based on the previous and
+the current HEAD of Origin that got pulled in with `git rebase`. All the steps above need to
```

Comment:
So the "downside" here we are trying to work around is that one rebase gets two changelog entries?